apps/cookbook/src/app/examples/chart-example/examples/yearly-overview-chart-example/yearly-overview-chart-example.component.ts

```diff
@@ -0,0 +1,158 @@
+import { Component, OnInit } from '@angular/core';
+import { Options, SeriesClickCallbackFunction, SeriesClickEventObject } from 'highcharts';
+
+import { ModalController, DesignTokenHelper } from '@kirbydesign/designsystem';
+
+const getColor = DesignTokenHelper.getColor;
+const fontSize = DesignTokenHelper.fontSize;
+
+function colorPoints(selectedYear: string) {
+  return function() {
+    var series = this.series;
+    for (var i = 0, ie = series.length; i < ie; ++i) {
+      var points = series[i].data;
+      for (var j = 0, je = points.length; j < je; ++j) {
+        if (points[j].graphic) {
+          if (points[j].category === selectedYear) {
+            points[j].graphic.element.style.fill = getColor('primary').value;
+          }
+
+          points[j].graphic.element.style.stroke = getColor('background-color').value;
+        }
+      }
+    }
+  };
+}
+
+@Component({
+  selector: 'cookbook-yearly-overview-chart-example',
+  templateUrl: './yearly-overview-chart-example.component.html',
+  styleUrls: ['./yearly-overview-chart-example.component.scss'],
+})
+export class YearlyOverviewChartExampleComponent implements OnInit {
+  height = 150;
+
+  private yearExpensesData = [0, 8761, 7760];
+  private currentTimeData = [0, 1000, 800];
+  private maxValue = Math.max(...this.yearExpensesData) + Math.max(...this.currentTimeData);
+  private years = [2018, 2019, 2020].reverse();
+  private yearsStr = this.years.map((year) => year.toString());
+  private selectedYear = this.yearsStr[0];
+  private lowerLimit = Math.max(...this.yearExpensesData) * 0.01;
+  // lower limit is shown as 2% of max value for UX reasons
+  private adjustedYearExpensesData = this.yearExpensesData.map((data) =>
+    this.lowerLimit >= data ? this.lowerLimit : data
+  );
+
+  private yearlyOverviewClick: SeriesClickCallbackFunction = (ev: SeriesClickEventObject) => {
+    this.modalController.showAlert({
+      title: 'Clicked chart',
+      message: 'You clicked on year: ' + ev.point.category,
+      okBtnText: 'Ok',
+    });
+
+    this.selectedYear = ev.point.category;
```

Comment:
Fix type warning:
```suggestion
    this.selectedYear = ev.point.category.toString();
```

Reply:
What type warning?

Reply:
![image](https://user-images.githubusercontent.com/9210691/81565108-1d5af100-9399-11ea-94de-0b8d02960ae7.png)
Seems like it is already string?